website/content/docs/commands/tls/index.mdx

```diff
@@ -0,0 +1,52 @@
+---
+layout: docs
+page_title: 'Commands: TLS'
+---
+
+# Nomad TLS
+
+Command: `nomad tls`
+
+The `tls` command is used to help with setting up a CA and certificates for Nomad TLS.
+
+## Basic Examples
+
+Create a CA:
+
+```shell-session
+$ nomad tls ca create
+==> Saved nomad-agent-ca.pem
+==> Saved nomad-agent-ca-key.pem
+```
+
+Create a client certificate:
+
+```shell-session
+$ nomad tls cert create -client
+==> Using nomad-agent-ca.pem and nomad-agent-ca-key.pem
+==> Saved dc1-client-nomad.pem
+==> Saved dc1-client-nomad-key.pem
+```
+
+For more examples, ask for subcommand help or view the subcommand documentation
+by clicking on one of the links in the sidebar.
+
+## Usage
+
+Usage: `nomad tls <subcommand> <subcommand> [options]`
+
+For the exact documentation for your Nomad version, run `nomad tls -h` to
+view the complete list of subcommands.
```

Comment:
We have versioned documented, so the version you're looking at here already should be the exact documentation.

```suggestion
```